<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasswdW</class>
 <widget class="QDialog" name="ChangePasswdW">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ChangePasswdW
{
border-image: url(:/new/prefix1/image/changepasswordimag/back_board.png);
}</string>
  </property>
  <widget class="QGroupBox" name="titleGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>980</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#titleGroupBox
{
border-image: url(:/new/prefix1/image/changepasswordimag/back_board1.png);
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="returnLockButton">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>27</y>
      <width>16</width>
      <height>26</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/image/changepasswordimag/return.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="titleLable">
    <property name="geometry">
     <rect>
      <x>394</x>
      <y>11</y>
      <width>191</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(76, 76, 76);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>更改密码</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/prefix1/image/changepasswordimag/ok.png)}
QPushButton:hover{border-image: url(:/new/prefix1/image/changepasswordimag/ok.png)}
QPushButton:pressed{border-image: url(:/new/prefix1/image/changepasswordimag/ok2.png)}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPasswd_lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/changepasswordimag/show.png);
color: rgb(208, 208, 208);</string>
   </property>
   <property name="text">
    <string>原密码  请输入6位数字</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPasswd_lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/changepasswordimag/show.png);
color: rgb(208, 208, 208);</string>
   </property>
   <property name="text">
    <string>新密码  请输入6位数字</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPasswd_lineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>400</y>
     <width>350</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/image/changepasswordimag/show.png);
color: rgb(208, 208, 208);</string>
   </property>
   <property name="text">
    <string>确认密码  请输入6位数字</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
